### PR TITLE
Update manifest and tooling

### PR DESCRIPTION
**Pull request by gregorylegarec**
Validating `konitor check` for:
* Linting
* Manifest
* Webpack config

### CODE REVIEW

File: manifest.konnector [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
+        "service": "Les données mises à disposition par Orange via ce connecteur contiennent la liste des films que vous avez visionnés à partir du 01/01/2015 en VOD payante ou gratuite (Replay TV) à partir de votre Livebox (à l’exception des contenus « adulte »).\n\n\nVous pourrez utiliser ces données dans différentes applications de votre Cozy, par exemple « La Musique de mes Films » (disponibilité prochaine sur le Cozy Store)"
+      },
       "permissions": {
         "accounts": {
           "description": "Utilisé pour récupérer les données de compte"
         }
       }
     }
   },
+  "manifest_version": "2",
```
Comment (CPatchane) [inline manifest.konnector:42]: This is a new field?

Comment (gregorylegarec) [reply manifest.konnector:42]: Yep !

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (doubleface) [inline .prettierrc.json:1]: eslint-config-cozy-app is supposed to bring this, no?

File: package.json [hunk 3]
```diff
@@ -24,7 +24,9 @@
   },
   "devDependencies": {
     "copy-webpack-plugin": "4.5.1",
+    "eslint": "4.19.1",
     "git-directory-deploy": "1.5.1",
+    "prettier": "1.12.1",
```
Comment (doubleface) [inline package.json:29]: I think prettier is not needed

File: webpack.config.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ module.exports = {
   plugins: [
     new CopyPlugin([
       { from: 'manifest.konnector' },
-      { from: 'package.json' },
```
Comment (doubleface) [inline webpack.config.js:14]: You need to keep package.json file. It is needed by the registry. At least I previouly go error from the registry when the version in package.json was not the same as the tag @CPatchane ?

**Comment by doubleface** [comment]
@gregorylegarec Sorry for the late review.

File: webpack.config.js (hunk 4 above)
Comment (CPatchane) [reply webpack.config.js:14]: It's required by the registry indeed.

File: .prettierrc.json (hunk 2 above)
Comment (gregorylegarec) [reply .prettierrc.json:1]: That's file is for the editor if I understand well. @CPatchane could you confirm please ?

File: package.json (hunk 3 above)
Comment (gregorylegarec) [reply package.json:29]: I think you're right, actually I think it makes no sens to add it as a dev-dependency if we not use it in a npm task to prettify. Could you confirm @CPatchane ?

File: webpack.config.js (hunk 4 above)
Comment (gregorylegarec) [reply webpack.config.js:14]: Well spotted, I will fix that and add a check in konitor.

File: .prettierrc.json (hunk 2 above)
Comment (CPatchane) [reply .prettierrc.json:1]: Indeed, we need this file to handle correctly prettier in Atom (except if you use eslint-prettier I think).

File: package.json (hunk 3 above)
Comment (CPatchane) [reply package.json:29]: I think if we put a `prettierrc.json` we should keep `prettier` as a dev-dependency, shouldn't we?

**Comment by gregorylegarec** [comment]
@doubleface @CPatchane I a take your remarks into account in this PR https://github.com/konnectors/cozy-konnector-orangevod/pull/32

**Pull request merged**